resolwe/flow/executors/docker/prepare.py

```diff
@@ -20,3 +23,31 @@ def post_register_hook(self):
         """Pull Docker images needed by processes after registering."""
         if not getattr(settings, 'FLOW_DOCKER_DONT_PULL', False):
             call_command('list_docker_images', pull=True)
+
+    def resolve_data_path(self, data=None, filename=None):
+        """Resolve data path for use with the executor.
+
+        :param data: Data object instance
+        :param filename: Filename to resolve
+        :return: Resolved filename, which can be used to access the given data
```

Comment:
Please, wrap the docstring at 72 characters.

Reply:
Should we have linters for these things? I never look at where I wrap things :-)

Reply:
It's not yet implemented in pycodestyle, but the PR @ https://github.com/PyCQA/pycodestyle/pull/674 seems on right track and that it will got approval from pycodestyle's maintainer.